assets/src/scripts/monitoring-location/components/hydrograph/select-actions.test.js

```diff
@@ -106,4 +110,51 @@ describe('monitoring-location/components/hydrograph/select-actions', () => {
         expect(testDiv.select(`#${TIME_FORM_ID}`).attr('hidden')).toBeNull();
         expect(testDiv.select(`#${DOWNLOAD_FORM_ID}`).attr('hidden')).toBe('true');
     });
-});
\ No newline at end of file
+
+    describe('monitoring-location/components/hydrograph/select-actions with no iv or gw data', () => {
```

Comment:
This isn't exactly what I meant by nesting describe sections. You are actually creating the select actions element twice for the nested describe because the beforeEach for the parent describe will also get executed for each test. I think it would be clearer to have:
```
describe('monitoring-location/components/hydrograph/select-actions', () => {
    #Define constants that are common but not TEST_STATE which will be different for the two sub-suites
    describe('Tests when iv or gw data is available', () => {
         #Add everything in the current describe except not the the nested describe
    describe('Tests with no IV or GW data', () => {
        # Add everything in the current nested describe.

Reply:
Ah, that makes so much more sense!! Thanks for pointing that out.